exercises/sublist/README.md

```diff
@@ -0,0 +1,22 @@
+# Sublist
+
+Given two lists determine if the first list is contained within the second list, if the second list is contained within the first list, if both lists are contained within each other or if none of these are true.
+
+Specifically, a list A is a sublist of list B if by dropping 0 or more elements from the front of B and 0 or more elements from the back of B you get a list that's completely equal to A.
+
+Examples:
+
+A = [1, 2, 3], B = [1, 2, 3, 4, 5], A is a sublist of B
+A = [3, 4, 5], B = [1, 2, 3, 4, 5], A is a sublist of B
+A = [3, 4], B = [1, 2, 3, 4, 5], A is a sublist of B
+A = [1, 2, 3], B = [1, 2, 3], A is equal to B
+A = [1, 2, 3, 4, 5], B = [2, 3, 4], A is a superlist of B
+A = [1, 2, 4], B = [1, 2, 3, 4, 5], A is not a superlist of, sublist of or equal to B
+
+* * * *
+
+For installation and learning resources, refer to the
+[exercism help page](http://exercism.io/languages/ruby).
+
```

Comment:
Should also include instructions about how to run the tests.

Reply:
I solved this problem once and for all by autogenerating READMEs: https://github.com/exercism/ruby/pull/721